docs/API.md

```diff
@@ -64,3 +64,29 @@ Acceptable formatting:
 
 ### The body of your document
 (required) A longer description of the topic, which will appear on its topic page. Must be 1,000 characters or less. Should not be the same as `short_description`. Some Markdown is allowed, such as links. Emoji are allowed.
+
+##  Making edits to a collection
+
+If you're making edits to a collection, the following fields are available for use. Not all fields are required.
+
+### items
+A comma-separated string containing any of the following values:
+
+- GitHub repository path (e.g. `defunkt/dotjs`)
+- GitHub username (e.g. `defunkt`)
+- GitHub organization (e.g. `github`)
+- Any web URL (e.g. `https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners`)
+- Any YouTube video URL (e.g. `https://www.youtube.com/watch?v=0fKg7e37bQE`)
+
+A complete `items` key-value pair might look like:
+
+`items: pybee/batavia, Homebrew/brew, https://www.youtube.com/watch?v=dSl_qnWO104`
```

Comment:
I would encourage allowing a list where each item is on a separate line. I think it makes the Markdown file more readable and I think it would also make it easier to parse when consuming the Markdown file.

Reply:
I think this line needs updating now that you can put items on separate lines.